peps/pep-0540.rst

```diff
@@ -66,7 +66,7 @@ simply that the locale is frequently misconfigured.  An obvious solution
 suggests itself: ignore the locale encoding and use UTF-8.
 
 
-Passthough for undecodable bytes: surrogateescape
+Passthrough for undecodable bytes: surrogateescape
 -------------------------------------------------
```

Comment:
```suggestion
--------------------------------------------------
```

To fix the build:
https://github.com/python/peps/actions/runs/10442226972/job/28914307478?pr=3909#step:5:138